src/components/ProgramList.tsx

```diff
@@ -0,0 +1,165 @@
+export const ProgramList: React.FC = () => {
+    // TODO remove test data and get new images
+    const imagesAndDescriptions: ProgramCardInfo[] = [
+        {
+            image: "https://i.imgur.com/UaEscmK.jpeg",
+            name: "Building Bridges with Music",
+            description:
+                "Children with special needs will be able to connect with the music teacher through an online video call to socialize and have fun while learning about music!",
+            startDate: "July 7, 2021",
+            endDate: "August 28, 2021",
+            format: "online",
+            tag: "art",
+        },
+        {
+            image: "https://i.kym-cdn.com/entries/icons/original/000/026/489/crying.jpg",
+            name: "MPM/JELIC",
+            description:
+                "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
+            startDate: "July 7, 2021",
+            endDate: "August 28, 2021",
+            format: "blended",
+            tag: "music",
+        },
+        {
+            image: "https://metro.co.uk/wp-content/uploads/2017/07/187144066.jpg?quality=90&strip=all&zoom=1&resize=644%2C428",
+            name: "Education Through Creativity",
+            description:
+                "Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.",
+            startDate: "July 7, 2021",
+            endDate: "August 28, 2021",
+            format: "online",
+            tag: "math",
+        },
+        {
+            image: "https://metro.co.uk/wp-content/uploads/2017/07/187144066.jpg?quality=90&strip=all&zoom=1&resize=644%2C428",
+            name: "Education Through Creativity",
+            description:
+                "Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.",
+            startDate: "July 7, 2021",
+            endDate: "August 28, 2021",
+            format: "online",
+            tag: "math",
+        },
+        {
+            name: "placeholder",
+            image: "",
+            description: "",
+            startDate: "",
+            endDate: "",
+            format: "",
+            tag: "",
+        },
+        {
+            name: "placeholder",
+            image: "",
+            description: "",
+            startDate: "",
+            endDate: "",
+            format: "",
+            tag: "",
+        },
+    ];
+
+    return (
+        <Center>
+            <Wrap spacing="50px" justify="space-between">
+                {imagesAndDescriptions.map((item, idx) => {
+                    return item.name == "placeholder" ? (
+                        <WrapItem
+                            flexBasis="300px"
+                            flexGrow={1}
+                            display="hidden"
+                            key={idx}
+                            _hover={{
```

Comment:
You might need to change the hover to be part of the inner card instead of the outer card so that it highlights the correct border if the inner card doesn't reach the full height
<img width="538" alt="Screen Shot 2021-07-23 at 12 13 43 AM" src="https://user-images.githubusercontent.com/42329517/126736654-819af52f-24f6-41b9-99b1-8217b3011908.png">

Reply:
What color should the hover be

Reply:
ok i put it on the inner box and used #0C53A0

![image](https://user-images.githubusercontent.com/50597354/126855303-7888e463-c99b-47ec-94b3-46ba8e3c1be6.png)